plutus-benchmark/README.md

```diff
@@ -1,52 +1,43 @@
 ## Plutus Benchmarks
 
-This directory contains three sets of benchmarks:
+This directory contains four sets of benchmarks:
 
 * `nofib`: Plutus versions of some of Haskell's `nofib` benchmarks from https://github.com/ghc/nofib.
 
    * The source for the individual programs is in `nofib/src`
    * There is an executable in `nofib/exe` which can be used to run the individual programs (compiled into Plutus)
    * The benchmarking code is in `nofib/bench`.
 
-   * To run the benchmarks using stack, type a command like this
-       * `stack bench plutus-benchmark:nofib` (run all benchmarks; this will take a long time)
-       * `stack bench plutus-benchmark:nofib --ba "clausify/formula2 -L300"` (run the `clausify/formula2`
+   * To run the benchmarks using cabal, type a command like this
+       * `cabal bench plutus-benchmark:nofib` (run all benchmarks; this will take a long time)
+       * `cabal bench plutus-benchmark:nofib --ba "clausify/formula2 -L300"` (run the `clausify/formula2`
```

Comment:
Doesn't seem to work:

```
$ cabal bench plutus-benchmark:nofib --ba "clausify/formula2 -L300"
Error: cabal: unrecognized 'bench' option `--ba'
```

And below.

Reply:
Oh thanks for catching this! I can't figure out how to get this to work but the `cabal run` corresponding command seems to work. So I'll just remove this one unless @kwxm or anyone knows how to get `bench` to work?

Reply:
It looks like you have to type things like 
```
cabal bench plutus-benchmark:nofib --benchmark-options "clausify/formula2 -L300"
```
As always, it took me some time to work out how to do that. I always get confused between how `cabal bench` and `cabal test` work.